### PR TITLE
PD-1941 dark mode form elements

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-1941](https://jira.mongodb.org/browse/PD-1941)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/tokens/src/typeScales.ts [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+
 const typeScales = {
   body1: {
-    fontSize: 13,
+    fontSize: BaseFontSize.Body1,
     lineHeight: 20,
   } as const,
   body2: {
-    fontSize: 16,
+    fontSize: BaseFontSize.Body2,
```
Comment (spark3-mdb) [inline packages/tokens/src/typeScales.ts:14]: 🔥🔥🔥

File: packages/typography/src/FormElements/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  [Mode.Dark]: css`
+    color: ${palette.gray.light1};
+  `,
+};
+
+type LabelProps = HTMLElementProps<'label', never> & {
+  darkMode?: boolean;
+  htmlFor: string;
+  disabled?: boolean;
+};
```
Comment (spark3-mdb) [inline packages/typography/src/FormElements/Label.tsx:47]: 🤷 Could we move these types for both Description and Label to `types.ts`?

**Review by spark3-mdb — CHANGES_REQUESTED** [review]
I like how you refactored here! The comments themselves are minor, but I'm not able to see the Typography page on Storybook. Is that just me?

Update: the page came back for me. I just had to build again.

File: packages/typography/src/FormElements/Label.tsx (hunk 2 above)
Comment (TheSonOfThomp) [reply packages/typography/src/FormElements/Label.tsx:47]: In principle, yes, but I feel like these interfaces are small enough that they can live here. If I were to move them I'd prefer to refactor each component into its own directory, for example `FormElements/Label/types.ts` etc

Comment (spark3-mdb) [reply packages/typography/src/FormElements/Label.tsx:47]: I like that idea of having separate directories! I'll leave the call up to you.

**Review by spark3-mdb — APPROVED** [review]

File: packages/typography/src/FormElements/Label.tsx (hunk 2 above)
Comment (TheSonOfThomp) [reply packages/typography/src/FormElements/Label.tsx:47]: Again, I think in principle yes, but for a small component like this I think it's easier to keep it all in one file. The threshold is blurry for when a component should be broken into multiple files

**Pull request merged**